magento/magento-cloud-docker
# MCLOUD-5856: Added the Healthcheck for DB and redis

Added the health check for DB container and Redis
     


### Description

https://jira.corp.magento.com/browse/MCLOUD-5856: 
Added Healthchecks for DB and Redis container.
### Fixed Issues (if relevant)

1. magento/magento-cloud-docker#116: Add additional health check for containers. 

### Manual testing scenarios

1. Run  ./vendor/bin/ece-docker build:compose --mode="developer"
2.  Run docker-compose up -d
3.  The container should be up and running with a healthy status.

### Contribution checklist

releasenotes
Added health checks for mySQL and Redis services.
